Bump actions/upload-artifact from 4.4.3 to 4.6.0 (#243)
rsksmart/devportal: rif/rns/libs/javascript/Getting-started.md

const rns = new RNS(web3)
```

> > If running the code in a webpage, don't need to copy the first two lines with the `import` statements, just instantiate them from the global scope.

Reviewer: ```suggestion
> Remember that if you are running the code in a webpage, no `import` statements are needed, just instantiate the libs made available in the global scope.
```
